docs/release-notes.txt

```diff
@@ -8,6 +8,17 @@
  Version 0.15
 **************
 
+Version 0.15.3
+==============
+
+**Release Date:** May 5, 2021
+
+**Bug Fixes**
+
+-  Images: Fix GPU support in CUDA 10.2 + Tensorflow 1.15 images.
+-  Trials: Fix trial environment dependency compatibility for
+   ``websockets>=9.0``.
+
```

Comment:
I forgot to add one for my fix, maybe `Trials: Fix a bug that caused trials to panic upon receiving too many rendezvous addresses` or something?
```suggestion
-  Trials: Fix trial environment dependency compatibility for
   ``websockets>=9.0``.
-  Trials: Trials: Fix a bug that caused trials to panic upon
    receiving too many rendezvous addresses.

```